packages/db/src/schema/index.ts

```diff
@@ -97,7 +97,8 @@ export const verificationTokens = mySqlTable(
 export const review = mySqlTable("review", {
   id: int("id").autoincrement().notNull().primaryKey(),
   userId: varchar("userId", { length: 255 }).notNull(),
-  restaurantId: int("restaurantId").notNull(),
+  restaurantId: varchar("restaurantId", { length: 50 }).notNull(),
+  name: varchar("name", { length: 255 }).notNull(),
```

Comment:
```suggestion
  restaurantName: varchar("name", { length: 255 }).notNull(),
```
I'd vote for this change for clarity.

Reply:
done